### DISCUSSION TITLE
Best sector time

### DISCUSSION THREAD

**Discussion by peerstream-zz** [f1multiviewer/issue-tracker#182] [Feature requests] [answered]
Hi,

I did see a similar topic however it concerned best lap times only so I created this one.

I would like to suggest to always show the **best sector time** performed by a driver under their current sector time, so that you can visually compare it to the current sector cleared out by a driver lap by lap. In addition, you could also show the calculated gap for each of the three sectors.

I know there are sector colors that indicate a driver's performance, however showing best sector timings allows you to have a numeric reference at hand.

Thank you for your wonderful app and your work.

**Comment by peerstream-zz** [comment]
Thank you so much for adding best sector times! And you did it in no time! 

Really appreciated 😄 

 From now on I will no longer have to write them down 😁

**Reply by f1multiviewer** [reply]
Haha best if you still do, it allows me to track them and ensures I don't forget 👍

**Comment by f1multiviewer** [comment, marked as answer]
As you said, fixed in the latest release!
